webpack/ForemanYupana/Components/Dashboard/DashboardActions.js

```diff
@@ -48,6 +49,27 @@ export const fetchLogs = () => async (dispatch, getState) => {
   }
 };
 
+export const getReportsQueue = () => async dispatch => {
+  try {
+    const {
+      data: { queue },
+    } = await API.get('uploads/queue');
+    dispatch({
+      type: YUPANA_GET_QUEUE,
+      payload: {
+        queue,
+      },
+    });
+  } catch (error) {
+    dispatch({
+      type: YUPANA_GET_QUEUE,
+      payload: {
+        queue: error.message,
```

Comment:
Does it make sense to put the error in a different field? This way we can be a bit more type safe.

Reply:
sure, 
by the way, do we want to inform the user if there is a network error or a problem with getting the process output? or should it be visible only in the terminal?

Reply:
Yes, there is a difference between unsuccessful upload process (which is sort of normal) as opposed to problem with getting data from our server (which is a problem).

Reply:
Thanks, now when there is an error retrieving the queue - it will show:
![Selection_171](https://user-images.githubusercontent.com/26363699/62053642-b3fe5d00-b220-11e9-8ed8-212405de1dcd.png)